### DISCUSSION TITLE
Is TypeScript required?

### DISCUSSION THREAD

**Discussion by DFazekas** [shadcn-ui/ui#680] [Q&A] [answered]
I have a vanilla React app scaffold using `create-react-app`, so it's using JavaScript, not TypeScript.

When following the installation for this UI kit (`npx shadcn-ui@latest init`), the command fails due to `Failed to load tsconfig.json. Couldn't find tsconfig.json`.

Is it possible to use this **without** TypeScript?

**Comment by dan5py** [comment]
Hi @DFazekas. Yes when using the CLI Typescript is required.
However you can use the manual installation in the docs and remove the types from the file, in order to have just JavaScript.

**Comment by elliot** [comment, marked as answer]
This is now supported: https://ui.shadcn.com/docs/changelog
